### ISSUE TITLE
Installer v3.99.60 fails

### ISSUE THREAD

**Issue by dmenken** [brianlala/AutoSPInstaller#20]
Using v3.99.60 on Windows Server 2016 attempting to install SP2016.
1. User accounts must be less than 29 characters or they fail to be verified in the beginning of the script.
2. I used the AutoSPSourceBuilder to download all the prerequisiteinstallerfiles but still get a "Could not locate source for .Net Framework 3.5.1" error listed in the script output.  Where can I download the correct file to drop in the folder?
3. When the script gets to the 'installing binaries' section, it always fails to locate the PIDkey in the XML file created by using the website creator and prompts for the key.  Then during the install it fails with several "SharePoint Foundation administration tool" errors.  At the end of the installer, the script fails with "Could not find SharePoint Server Setup log file!" and stops.

Any help is appreciated.  Used AutpSPInstaller back on 2010 and loved it.

**Comment by dmenken** [comment]
Looking for clarification on the path structure.  The 'Guide.docx' lists the path as "\temp\sp\..." and then as "\sp\..." but the 'AutoSPInstallerFolderStructure.txt' file shows the path as "\sp\...".  Is there a preferred structure?  Can this be on any drive letter as the docx file shows D:\temp but I'm using H:\sp and have tried H:\temp\sp.

**Comment by dmenken** [comment]
I must be the only person still using this script ...
1. The path issue is mute as the script handles it.  I used the \SP\... path structure.
2. The issue of the PIDkey was caused by the Online XML builder.  It does not store the PIDkey in the correct format and in the correct location.  I created a copy of the PIDkey in the Configuration node as per "https://technet.microsoft.com/en-us/library/cc287749.aspx" and it worked. No longer prompted during binary install.
3. Online XML creator didn't add the line to create logs so the script always failed.  Added the Logging attribute in the Configuration node per "https://technet.microsoft.com/en-us/library/cc287749.aspx" file format to resolve.
4. The SETUP.EXE errors in "SharePoint Foundation administration tool" seems to have been created by the slip-streamed install path.  I removed all the files in the "\sp\2016\SharePoint\updates" from "16.0.4627.1000 (December 2017)" and it installs without issue.

Still working through some additional minor issues but I got it to run all the way through the script.
